cradle-core/build.gradle

```
@@ -1,6 +1,8 @@
 dependencies {
-	compile "org.slf4j:slf4j-api:${slf4j_version}"
-	compile "org.slf4j:slf4j-log4j12:${slf4j_version}"
+	compileOnly "org.slf4j:slf4j-api:${slf4j_version}"
+	compileOnly "org.slf4j:slf4j-log4j12:${slf4j_version}"
+	testImplementation "org.slf4j:slf4j-api:${slf4j_version}"
+	testImplementation "org.slf4j:slf4j-log4j12:${slf4j_version}"

```

If you want to allow the projects that uses cradle as a library declare the actual implementation of logging you should declare them like this
```suggestion
	implementation "org.slf4j:slf4j-api:${slf4j_version}"
	testImplementation "org.slf4j:slf4j-log4j12:${slf4j_version}"
```